.github/workflows/release-image.yml

```diff
@@ -44,8 +43,13 @@ jobs:
         run: |
           docker tag ${{ env.OPERATOR_IMAGE }} ${{ env.RH_OPERATOR_IMAGE }}
           make operator-image-push OPERATOR_IMAGE=${{ env.RH_OPERATOR_IMAGE }}
+      - name: Create release
+        uses: softprops/action-gh-release@v1
+        with:
+          files: |
+            storageos-operator.yaml
```

Comment:
Are these the right names given we are in the context of the api-manager repo?

Reply:
Nope, I was meant to delete those lines 😓

Reply:
Should be fixed now